public/controllers/agent/components/agents-preview.js

```diff
@@ -236,7 +236,7 @@ export const AgentsPreview = compose(
                               <EuiToolTip
                                 position='top'
                                 content='View agent details'>
-                                  <a onClick={() => 
+                                  <a onClick={() =>
                                   this.props.tableProps.showAgent(
                                     this.lastAgent
                                   )}>{this.lastAgent.name}</a>
```

Comment:
Can you separate this on a new arrow function method?

Reply:
Yes, there I do, thanks for the observation +1